tests/storage/cdi_upload/test_upload_virtctl.py

```
@@ -53,6 +55,31 @@ def skip_no_reencrypt_route(upload_proxy_route):
         pytest.skip("Skip testing. The upload proxy route is not re-encrypt.")
 
 
+@pytest.fixture(scope="function")
```

Ruff (0.15.0)

[warning] `scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)